tools/log_rotate/install_script.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Create directory, which keeps all logrotate-config to rotate hourly.
+mkdir /etc/logrotate.hourly.d
+
+cat << EOF | tee /etc/logrotate.hourly.conf
+# packages drop hourly log rotation information into this directory
+include /etc/logrotate.hourly.d
+EOF
+
+chmod 644 /etc/logrotate.hourly.conf
+
+cat << EOF | tee /etc/cron.hourly/logrotate
+#!/bin/bash
+test -x /usr/sbin/logrotate || exit 0
+/usr/sbin/logrotate /etc/logrotate.hourly.conf
+EOF
+
+chmod 775 /etc/cron.hourly/logrotate
+
+# Logrotate configuration to rotate gcsfuse logs.
+cat << EOF | tee /etc/logrotate.hourly.d/gcsfuse
+/var/log/gcsfuse.log {
+  rotate 10
+  size 5G
+  missingok
+  notifempty
+  compress
+  dateext
+  dateformat -%Y%m%d-%s
+  delaycompress
+  sharedscripts
+  postrotate
+	  systemctl kill -s HUP rsyslog.service >/dev/null 2>&1 || true
+  endscript
+}
+EOF
+
+# Make sure gcsfuse-logrotate config got placed correctly.
+if [ $? -eq 0 ]; then
+        echo "Logrotate config for gcsfuse updated successfully!"
+else
+        echo "Please install linux package - logrotate"
+        exit 1
+fi
+
+# Syslog configuration to filter and redirect the logs from /var/log/syslog to
+# /var/log/gcsfuse.log.
+cat > /etc/rsyslog.d/08-gcsfuse.conf <<EOF
```

Comment:
just for my understanding - why is specifically 08 here?

Reply:
This number is for config file ordering and precedence. I.e. if the same parameter configuration is present in 10-x.conf and 20-y.conf the latter will overwrite first one. 08 is just a random number, we can increase this number.

Reply:
we can write a comment for this.